### PR TITLE
[jb] Add support for middle button/ctrl/cmd click to open Oozie related links in a new tab

### PR DESCRIPTION
**Pull request by JohanAhlen**
Throughout the Job Browser we use click events on table rows and because they're not anchor elements a ctrl/cmd click or middle mouse button click on the row won't open it in a new tab.

The proper solution would be to "modernize" this entire code-line but to solve the issue short term I've added certain event checks to detect the intention to open in a new tab.

This also takes care of some broken links where only the context popover were shown instead of document when clicking. Now the context popover is only shown when clicking the 'i' next to the link.

Tested manually.

### CODE REVIEW

File: apps/jobbrowser/src/jobbrowser/templates/job_browser.mako [hunk 1]
```diff
@@ -3048,6 +3091,22 @@ ${ commonheader("Job Browser", "jobbrowser", user, request) | n,unicode }
           });
         }
       }
+
+      self.onTableRowClick = function (event, id) {
+        var openInNewTab = event && (event.which === 2 || event.metaKey || event.ctrlKey);
```
Comment (bjornalm) [inline apps/jobbrowser/src/jobbrowser/templates/job_browser.mako:3096]: nit: 
Could make it a bit easier to understand with `var middleMouseButton = 2`

File: apps/jobbrowser/src/jobbrowser/templates/job_browser.mako [hunk 2]
```diff
@@ -3048,6 +3091,22 @@ ${ commonheader("Job Browser", "jobbrowser", user, request) | n,unicode }
...
+        var idToOpen = id || self.id();
+        if (idToOpen && idToOpen !== '-') {
```
Comment (bjornalm) [inline apps/jobbrowser/src/jobbrowser/templates/job_browser.mako:3098]: nit: what does the id value `-` represent?

**Review by bjornalm — APPROVED** [review]
Nice work

File: apps/jobbrowser/src/jobbrowser/templates/job_browser.mako (hunk 2 above)
Comment (JohanAhlen) [reply apps/jobbrowser/src/jobbrowser/templates/job_browser.mako:3098]: No clue haha, it was there in previous checks (line 2076)

File: apps/jobbrowser/src/jobbrowser/templates/job_browser.mako (hunk 1 above)
Comment (JohanAhlen) [reply apps/jobbrowser/src/jobbrowser/templates/job_browser.mako:3096]: Good one, I'll add it.

**Review by nidhibhatg — COMMENTED** [review]
LGTM

**Review by nidhibhatg — APPROVED** [review]

**Pull request merged**